### ISSUE TITLE
Can not reproduce results on the Fishscapes static dataset

### ISSUE THREAD

**Issue by gaozhitong** [shjung13/Standardized-max-logits#7]
Hi, thanks for your contribution! I am currently having trouble on reproducing the reported results on the Fishscapes static dataset.

I use the offered pre-trained model "r101_os8_base_cty.pth" and **can get the exactaly same results on the Fishscapes lost & found as reported in the paper** and roughly same results on the Road Anomaly dataet (difference < 1%). However, when I simply change the "ood_dataset_path" to Fishscapes static dataset. I get the following statistics, which are different from the reported values in the paper. Could you please kindly explain this?

`AUROC score: 0.9547932440646849
AUPRC score: 0.51792152018834
FPR@TPR95: 0.2243785053302718`

**Comment by shjung13** [comment]
Hello @gaozhitong,

Thank you for your interest in our work!

There are several versions of the Fishyscapes static dataset, and if I remember correctly, we used version 3.

Please check your version of the static dataset, and let me know if you cannot reproduce the results with the same version.

Thank you.

**Comment by gaozhitong** [comment]
Got it! Thanks for your quick reply. I will check with the static dataset with version 3.

**Comment by GewelsJI** [comment]
Hi, @gaozhitong 

Have you done that? Could you share your experience of how to prepare static dataset with version 3?
